Reply/app/src/main/java/com/example/reply/ui/theme/Theme.kt

```diff
@@ -15,43 +15,90 @@ import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.platform.LocalView
 import androidx.core.view.ViewCompat
 
-private val DarkColorScheme = darkColorScheme(
-    primary = Purple80,
-    secondary = PurpleGrey80,
-    tertiary = Pink80
+//Material 3 color schemes
```

Comment:
Hmm I actually think the color schemes should go in `Color.kt`

Reply:
I think it's fine to be in theme because default project also generate colorSchemes in Theme.kt and Color.kt just contains color codes for reference. Many of our samples follows this as well.

Reply:
Okay, sounds good then